components/atom/tag/src/Actionable/Container.js

```diff
@@ -4,9 +4,16 @@ import PropTypes from 'prop-types'
 /**
  * Component treated as an anchor when href is defined
  */
-const ActionableTagContainer = ({Link, href, target, children, ...props}) => {
+const ActionableTagContainer = ({
+  Link,
+  href,
+  target,
+  rel,
```

Comment:
Maybe a default value for `rel` can be`"tag"` ,  because it's `atom/tag`. What do you think?

Reply:
is there the necessity to add different values for `rel`other than `tag`?